Стр/9.1/9.1_1.py

```diff
@@ -1,3 +1,5 @@
 """ \n этот символ используется в троках для вывода многострочного результата """
 
 # TODO: приведите примеры
+print(" A \n B ")
+print(' A \n B \n C ')
```

Comment:
`````suggestion
print(" A \n B ")
print(' A \n B \n C ')
```
`"""А можно и так"""
print(' A \n \n B ')`